docs/DATA_PRIVACY.md

```diff
@@ -0,0 +1,24 @@
+# Data Privacy
+
+Please familiarise yourself with the following information:
+
+- [eLife's Privacy Notice](https://elifesciences.org/privacy)
+
+## Overview
+
+`elife-xpub` has been designed to integrate with [eLife's journal site](elifesciences.org).
+As such users control and manage the privacy of their data via their [ORCiD](orcid.org) account.
+eLife ensures that user data is kept up-to-date via the [Profiles Service](https://github.com/elifesciences/profiles)
+and its this service that `elife-xpub` interacts with to get all the user data.
+
+## Review 2019-01-22
+
+Peter, Hannah reviewed the handling of data within `elife-xpub` as a precursor to launch.
```

Comment:
this looks like a document with legal value (GDPR?), hence you may need surnames too?

Reply:
I think I will change this to "Product Manager" and "Technical Lead" ?